TsLint.yml

```diff
@@ -0,0 +1,16 @@
+sniffs:
```

Comment:
Is this the default name for the configuration file? If not; I'd prefer it to be placed in `config/`.

Reply:
Could do that. `config` is a special folder setup by TYPO3. Right now there are site configurations.